### DISCUSSION TITLE
Dominican channels

### DISCUSSION THREAD

**Discussion by Hughg111** [iptv-org/community#585] [Q&A]
Looking for a latin/spanish only Iptv service that has channels from DR.Thanks for any help

**Comment by Pecaquito** [comment]
Trata esta colecion que tengo a lo major te guste, coloca este URL o link en iptv player, te recomiendo que use, OTT TV, app, o OTT Navigator.
Lista, Url: https://bit.ly/3nIXReF
OTT NAVIGATOR: Version 1.6.6.9
https://bit.ly/OTT1691N
Código De Configuration: BRz5RP
Para cargar el Código De configuration, seleciona, setting=configuracion, luego, extending settings=cofig extendida, backup config, loud previously config, ingresa este codigo, apply, y seleciona everything, Espera un minuto, cierra LA app y la abre de Nuevo.
